plat/rockchip/rk3399/drivers/pmu/pmu.c

```diff
@@ -249,11 +667,20 @@ static int cores_pwr_domain_on(unsigned long mpidr, uint64_t entrypoint)
 	return 0;
 }
 
-static int cores_pwr_domain_off(void)
+static int cores_pwr_domain_off(uint32_t lvl, plat_local_state_t lvl_state)
```

Comment:
It's confusing that you have 2 different functions for suspend (`cores_pwr_domain_suspend()` and `hlvl_pwr_domain_suspend()`) but 1 combined function for off (`cores_pwr_domain_off()`). Similarly for `cores_pwr_domain_on()`.